collect_app/build.gradle

```diff
@@ -10,6 +10,29 @@ com.android.ddmlib.DdmPreferences.setTimeOut(60000)
 ant.condition(property: "os", value: "windows") { os(family: "windows") }
 ant.condition(property: "os", value: "unix" ) { os(family: "unix") }
 
+// Build numbers were manually set until 1063
+def LEGACY_BUILD_NUMBER_OFFSET = 1063
+
+def getMasterCommitCount = { ->
```

Comment:
Add a quick reference to StackOverflow so someone can get more context. E.g.:
// Based on http://stackoverflow.com/questions/17097263#24121734